### DISCUSSION TITLE
How can I use iframe?

### DISCUSSION THREAD

**Discussion by imnithish** [varabyte/kobweb#387] [Q&A] [answered]
So, I wanted to implement Google map and a marker. I'm a rookie in web dev.
Basically, I wanted to implement as mentioned like [this](https://forum.bootstrapstudio.io/t/google-map-marker-with-no-api-key/6514). How can I do this with Kobweb?

**Comment by bitspittle** [comment, marked as answer]
I haven't used Iframes myself, but there's an Iframe component inside Compose HTML.

Maybe try something like this?

```kotlin
Iframe(
  attrs = {
    attr("width", "100%")
    attr("height", "500")
    attr("src", "..."))
  }
)
```
